generators/app/templates/test/_library_/qunit/testsuite.qunit.js

```diff
@@ -20,7 +20,11 @@ sap.ui.define(function () {
 				qunitBridge: true,
 				useFakeTimers: false
 			},
-			module: "./{name}.qunit"
+			module: "./{name}.qunit",
+			coverage: {
+				only: ["<%= libURI %>"],
+				never: ["test-resources/<%= libURI %>"]
```

Comment:
```suggestion
				never: ["test-resources/"]
```
Could be simplified by never instrumenting test-resources modules in general

Reply:
done